index.js

```diff
@@ -38,13 +40,6 @@ export default class OTPInputView extends Component {
         }
     }
 
-    componentWillReceiveProps(nextProps) {
```

Comment:
@hasithaWalpola what about this code? I think it should remain in the library.

Reply:
yes, it should remain